### PR TITLE
Form styling

### PR DESCRIPTION
**Pull request by afacey**
- styled autocomplete suggestions to appear as a dropdown of the user library input
- styled the radio buttons and submit button to look like the wireframe

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -6,4 +6,127 @@ html { -moz-box-sizing: border-box; -webkit-box-sizing: border-box; box-sizing:
...
+
+.autoCompleteResults button {
+  border: 0;
+  background-color: #EEE;
+  padding: 10px 0 10px 10px;
+  width: 100%;
+  text-align: left;
+  transition: background-color 0.3s;
+}
+
```
Comment (connietee22) [inline src/App.css:156]: Amazing drop-down! 👏

**Review by connietee22 — APPROVED** [review]
Looks goooood to go!

**Comment by kdlry** [comment]
Good to go! Nice job

File: src/App.css (hunk 1 above)
Comment (raphaelroques) [reply src/App.css:156]: good job !! rdy to merge !

**Review by kdlry — APPROVED** [review]
Approved!

**Pull request merged**